hooks/pre/29-remove_package_httpd_conf.rb

```diff
@@ -1,10 +1,13 @@
-# Some packages like pulp place configuration files in /etc/httpd/conf.d that
+# Pulp places a configuration file in /etc/httpd/conf.d that
 # contain duplicates declarations of directives that the puppetlabs-apache
-# module configured elsewhere.
+# module configured elsewhere. This is only relevant on smart proxies.
+# The main katello install overwrites the pulp.conf.
 #
 # * pulp places a pulp.conf that contains a duplicate WSGI 'pulp'
 #   daemon that 05-pulps-https.conf contains.
 #
-%w(pulp.conf).each do |file|
-  File.delete(File.join("/etc/httpd/conf.d/", file)) if File.file?(File.join("/etc/httpd/conf.d/", file))
+if Kafo::Helpers.module_enabled?(@kafo, 'foreman_proxy_content') && !app_value(:noop)
```

Comment:
Ah sorry, mistake on my part - can you fix this to say `!Kafo::Helpers.module_enabled?(@kafo, 'katello')` instead?

foreman_proxy_content is enabled on proxies and katellos, instead check if it's not a katello.